packages/teuchos/parameterlist/src/Teuchos_ParameterList.hpp

```diff
@@ -811,6 +811,21 @@ TEUCHOSPARAMETERLIST_LIB_DLL_EXPORT bool haveSameValues( const ParameterList& li
     bool verbose = false);
 
 
+/** \brief Returns true if two parameter lists have the same values independent of ordering.
+ *
+ * Two parameter lists may have the same values but may not be identical.  For
+ * example, two parameters can have the same values but not have the same
+ * documentation strings or the same validators.
+ *
+ * \note This test does not respect ordering of the ParameterList entries; the same values in a different
```

Comment:
Should this not say "this function" instead of "this test"?

Reply:
I've switched "function" to "test" in both `haveSameValues" and "haveSameValuesSorted".